_topic_maps/_topic_map_rosa.yml

```diff
@@ -301,9 +301,9 @@ Topics:
     File: persistent-storage-csi
   - Name: Managing the default storage class
     File: persistent-storage-csi-sc-manage
-  - Name: AWS Elastic Block Store CSI Driver Operator
+  - Name: Amazon Elastic Block Store CSI Driver Operator
     File: persistent-storage-csi-ebs
-  - Name: AWS Elastic File Service CSI Driver Operator
```

Comment:
In the product (Operator Hub), it is called as AWS EFS CSI driver Operator. Wouldn't changing here w/o changing the operator name confuse?

Reply:
I have revoked the instance here and also on Line 304.